packages/connect/src/protocol-grpc/request-header.spec.ts

```diff
@@ -34,7 +34,7 @@ describe("requestHeader", () => {
       "user-agent",
     ]);
     expect(headers.get("Content-Type")).toBe("application/grpc+proto");
-    expect(headers.get("User-Agent")).toMatch(/^connect-es\/\d+\.\d+\.\d+/);
+    expect(headers.get("User-Agent")).toMatch(/^connect-es\/.*/);
```

Comment:
Let's not modify this test. It makes sure the default user agent doesn't regress.